README.md

```diff
@@ -107,8 +107,8 @@ It is still possible to prove membership and equality in the refinement logic,
 even though the categorical judgements are inaccessible. In order to do this,
 an equality type is introduced, whose *truth* is defined specifically so as to
 coincide with the (mere) evidence of CTT's equality judgement. In practice, you
-will often see goals like `H >> M in A [ext •]`, and in this case you are
```

Comment:
This one is not meant to be \in; It is perhaps an unfortunate notation, but I am using "in" for the type/proposition, and "\in" for the judgement.